WcaOnRails/app/views/competitions/_my_competitions_table.html.erb

```diff
@@ -0,0 +1,54 @@
+<% if competitions.length < 1 %>
+  <% message = upcoming ? "You currently have no upcoming competitions! Check out the #{link_to 'competitions list', '/results/competitions.php'}.".html_safe : "You have no past competitions." %>
```

Comment:
I don't think there's any value to making message a variable here. Can't you just do an if statement inside of erb inside of the `alert` right below?